Configure

```diff
@@ -812,6 +824,16 @@ if ($libs =~ /(^|\s)-Wl,-rpath,/
 	"***** any of asan, msan or ubsan\n";
 }
 
+if (scalar(@seed_sources) == 0) {
+    print "Using implicit seed configuration\n";
+    push @seed_sources, 'os';
+}
+die "Cannot seed with none and anything else"
+    if scalar(grep('none', @seed_sources)) > 0 && scalar(@seed_sources) > 1;
```

Comment:
That grep won't work, or rather, it will match *everything* (because `'none'` is an expression that always evaluates true, like all non-empty strings).

If you want to match exact string, this form is better: `grep { $_ eq 'none' } @seed_sources`